release/notes.py

```diff
@@ -0,0 +1,172 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
All the other files have an empty line first, should we make this one similar too?